src/fides/api/api/v1/endpoints/privacy_preference_endpoints.py

```diff
@@ -187,6 +189,30 @@ def get_request_origin_and_config() -> Tuple[Optional[str], Optional[str]]:
     )
 
 
+def get_ip_address(request: Request) -> Optional[str]:
+    """Get client ip, preferring x-forwarded-for if it exists, otherwise, dropping back to
+    request.client.host"""
+    x_forwarded_for = (
+        request.headers.get("x-forwarded-for") if request.headers else None
+    )
+
+    logger.info(f"Temporary log: X-Forwarded-For {x_forwarded_for}")
+
+    client_ip: Optional[str] = None
+    if x_forwarded_for:
+        try:
+            client_ip = x_forwarded_for.split(",")[0].strip()
+        except AttributeError:
+            pass
```

Comment:
probably missing something obvious but what's the `AttibuteError` that can be thrown here? is it possible that the `x_forwarded_for` variable isn't a string?

Reply:
I was just thinking if it didn't exist?